src/patient-getter/patient-getter.resource.ts

```diff
@@ -1,38 +1,36 @@
-import { openmrsFetch } from "@openmrs/esm-framework";
-
+import useSWR from "swr";
+import { fhirBaseUrl, openmrsFetch } from "@openmrs/esm-framework";
 /**
- * This is a somewhat silly resource function. It searches for a patient
- * using the REST API, and then immediately gets the data using the FHIR
- * API for the first patient found. OpenMRS API endpoints are generally
- * hit using `openmrsFetch`. For FHIR endpoints we use the FHIR API
- * object.
+ * This hook searches for a patient using the provided search term from the OpenMRS FHIR API. It leverages the [useSWR] hook from the SWR library to fetch data. SWR provides a number of benefits over the standard React [useEffect] hook:
```

Comment:
Is unlimited line length the new norm? Not sure I love it

Reply:
It shouldn't be